pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -326,38 +339,101 @@ func indentLines(indent string, lines ...string) []string {
 	return ret
 }
 
-func describeProjectAndServer(project *projectapi.Project, server string) string {
-	if server != "" {
+type formatter interface {
+	ResourceName(obj interface{}) string
+}
+
+func namespaceNameWithType(resource, name, namespace, defaultNamespace string, noNamespace bool) string {
+	if noNamespace || namespace == defaultNamespace || len(namespace) == 0 {
+		return resource + "/" + name
+	}
+	return resource + "/" + name + "[" + namespace + "]"
```

Comment:
> Talked with @liggitt, I think we should do resource.group[namespace]/name. Doing it that way makes

I'm still not sold on namespace at the end due to sorting problems even if it makes copy/paste easier.